int64 mode
---

int64 error_code